packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/TablesViewer.tsx

```diff
@@ -110,7 +114,11 @@ export function TablesViewer({ table, query, isLiveQuery }: Props) {
   });
 
   return (
-    <div className="!-mt-10 space-y-4">
+    <div
+      className={cn("space-y-4", {
+        "!-mt-10": indexer.type === "hosted",
```

Comment:
what does the `!-` prefix mean?

Reply:
`!important`. I've used it here because it clashes with `space-y-4`. I know generally `!important` is a bad practice but in this case it's only isolated to this `div` due to layout "exception" that happens when SQL editor is not shown when local indexer is used.